Undefined type errors in index.d.ts

**Describe the bug**
When attempting to use this library in a typescript project it will compile with a number of errors:

```javascript
ERROR in node_modules/@ffmpeg/ffmpeg/src/index.d.ts:1:43 - error TS2304: Cannot find name 'FS'.
    1 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:1:56 - error TS2304: Cannot find name 'FS'.
    1 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:1:109 - error TS2538: Type 'symbol' cannot be used as an index type.
    1 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:1:122 - error TS2304: Cannot find name 'FS'.
    1 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:2:64 - error TS2304: Cannot find name 'FS'.
    2 type FSMethodArgs = { [K in FSMethodNames]: Parameters<(typeof FS)[K]> };

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:3:66 - error TS2304: Cannot find name 'FS'.
    3 type FSMethodReturn = { [K in FSMethodNames]: ReturnType<(typeof FS)[K]> };

    node_modules/@ffmpeg/ffmpeg/src/index.d.ts:105:42 - error TS2591: Cannot find name 'Buffer'. Do you need 
to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your 
tsconfig.
    105 export function fetchFile(data: string | Buffer | Blob | File): Uint8Array;
```
**Expected behavior**
index.d.ts should compile without any errors.

**Desktop:**
 - OS: Windows
 - Typescript 3.8.3

**Additional context**
Running from within a clean Angular project.
Despite what the error message above suggests, @types/node is already installed.
Using `right click -> Go to definition` on Buffer on line 105 successfully finds it in VS Code, but typescript does not seem to know about it. No definition can be found for `FS`, however.

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,6 +1,6 @@
 export const FS: {
     writeFile: (fileName: string, binaryData: Uint8Array) => void,
-    readFile: (fileName: string) => void,
+    readFile: (fileName: string) => Uint8Array,
     unlink: (fileName: string) => void,
 }
 

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,3 +1,9 @@
+export const FS: {
+    writeFile: (fileName: string, binaryData: Uint8Array) => Promise<void>,
+    readFile: (fileName: string) => Promise<Uint8Array>,
+    unlink: (fileName: string) => Promise<void>,
+}
+
 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];
 type FSMethodArgs = { [K in FSMethodNames]: Parameters<(typeof FS)[K]> };
 type FSMethodReturn = { [K in FSMethodNames]: ReturnType<(typeof FS)[K]> };

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,7 +1,7 @@
 export const FS: {
-    writeFile: (fileName: string, binaryData: Uint8Array) => Promise<void>,
-    readFile: (fileName: string) => Promise<Uint8Array>,
-    unlink: (fileName: string) => Promise<void>,
+    writeFile: (fileName: string, binaryData: Uint8Array) => void,
+    readFile: (fileName: string) => void,
+    unlink: (fileName: string) => void,
 }
 
 type FSMethodNames = { [K in keyof typeof FS]: (typeof FS)[K] extends (...args: any[]) => any ? K : never }[keyof typeof FS];

